docs/current/guides/757394-use-service-containers.md

```diff
@@ -205,136 +213,111 @@ When a service is bound to a container, it also conveys to any outputs of that c
+
+Here is an example of how to instantiate host services with Dagger:
 
 <Tabs groupId="language" className="embeds">
 <TabItem value="Go">
 
-<Embed id="JwuCvswjsEM" />
-
-Here's what happens on the last line:
-
-1. The client requests the `ping` container's stdout, which requires the container to run.
-1. Dagger sees that the `ping` container has a service binding, `redisSrv`.
-1. Dagger starts the `redisSrv` container, which recurses into this same process.
-1. Dagger waits for health checks to pass against `redisSrv`.
-1. Dagger runs the `ping` container with the `redis-srv` alias magically added to `/etc/hosts`.
+<Embed id="" />
 
 </TabItem>
 <TabItem value="Node.js">
 
-<Embed id="WRo9QMK9GKZ" />
-
-Here's what happens on the last line:
-
-1. The client requests the `ping` container's stdout, which requires the container to run.
-1. Dagger sees that the `ping` container has a service binding, `redisSrv`.
-1. Dagger starts the `redisSrv` container, which recurses into this same process.
-1. Dagger waits for health checks to pass against `redisSrv`.
-1. Dagger runs the `ping` container with the `redis-srv` alias magically added to `/etc/hosts`.
+<Embed id="" />
 
 </TabItem>
 <TabItem value="Python">
 
-<Embed id="vtG-PyKz2E5" />
-
-Here's what happens on the last line:
-
-1. The client requests the `ping` container's stdout, which requires the container to run.
-1. Dagger sees that the `ping` container has a service binding, `redis_srv`.
-1. Dagger starts the `redis_srv` container, which recurses into this same process.
-1. Dagger waits for health checks to pass against `redis_srv`.
-1. Dagger runs the `ping` container with the `redis-srv` alias magically added to `/etc/hosts`.
+<Embed id="" />
 
 </TabItem>
 </Tabs>
 
-:::note
-Dagger cancels each service run after a 10 second grace period to avoid frequent restarts.
-:::
+### Container services to the host
+Starting with Dagger v0.8.8, you can client containers to the host.
 
-It's worth noting that services are just containers, and all containers in Dagger have run-exactly-once semantics. Concurrent runs of the same container synchronize and attach to the same run with progress/logs multiplexed to each caller. Canceling a run only interrupts the container process when all runs are canceled, so new clients can come and go throughout a service run.
+This type of container networking is useful when you need a client container to be able to access services running on the host. 
 
-Run-exactly-once semantics are very convenient. You don't have to come up with names and maintain instances of services; they're content-addressed, so you just use them by value. You also don't have to manage the state of the service; you can just trust that it will be running when needed and stopped when not.
+Call `Host.service([]PortForward)` to create a service that proxies traffic through the host to the configured ports. You then set the service binding on the client container to the host.
```

Comment:
I think this is Go-specific syntax? It would be better to have this in the Go tab and make this more generic so it applies to all SDKs equally.

Reply:
It could be simplified to `Host.service()` - not sure what our current convention is for language-neutral Object.method references.